_posts/for-developers/apps/first-app/2019-01-01-05_writing-data.md

```diff
@@ -21,10 +21,10 @@ async function addNote(note, notesList) {
   newNote.addRef(rdf.type, schema.TextDigitalDocument);
 
   // Set the Subject's `schema:text` to the actual note contents:
-  newNote.addLiteral(schema.text, note);
+  newNote.addString(schema.text, note);
```

Comment:
I don't understand the implications of this.

Reply:
This was an API change in Tripledoc, and updates the documentation to match it.